src/Environment.js

```diff
@@ -1,12 +1,38 @@
-const {
-    Environment,
-    Network,
-    RecordSource,
-    Store,
-  } = require('relay-runtime');
+import {Environment, Network, RecordSource, Store} from 'relay-runtime';
+import {SubscriptionClient} from 'subscriptions-transport-ws'
+import 'whatwg-fetch'
```

Comment:
Prefer isomorphic fetch

Reply:
agreed

Reply:
`Error: Can't find variable: self`
I will go back on this further.